tools/Scripts/SetupRepository.ps1

```diff
@@ -1,28 +1,27 @@
 param(
-    [switch] $iOS
+    [switch] $iOS,
+    [switch] $NoDownloads
 )
 
-. "$PSScriptRoot\SymbolicLinkHelpers.ps1"
-. "$PSScriptRoot\ExternalDependencyHelpers.ps1"
+. "$PSScriptRoot\SetupRepositoryFunc.ps1"
```

Comment:
If the *Func.ps1 files contain only shared functions perhaps they should be renamed to *.psm1 and included with Import-Module?